.github/workflows/build_pr_documentation.yml

```diff
@@ -0,0 +1,119 @@
+name: Build PR Documentation
+
+on:
+  workflow_call:
+    inputs:
+      commit_sha:
+        required: true
+        type: string
+      pr_number:
+        required: true
+        type: string
+      package:
+        required: true
+        type: string
+      path_to_doc:
+        type: string
+
```

Comment:
in [transformers pr_doc](https://github.com/huggingface/transformers/blob/master/.github/workflows/build_dev_documentation.yml#L6-L8), there are lines about concurrency

```py
concurrency:
  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
  cancel-in-progress: true
```

do we need them here as well?

Reply:
This should be on the workflows that apply this template (if the library wants it), see for instance in [this PR](https://github.com/huggingface/doc-builder/pull/138) (GitHub doesn't let me link to a specific line but it's line 6 in the build_pr_documentation file)